### ISSUE TITLE
OpenSSL 3.0.7 is required

### ISSUE THREAD

**Issue by Ismoh** [Ismoh/noita-vscode-debugger#1]
Horscht:
>finally got it to work by copying the DLLs from <https://wiki.overbyte.eu/arch/openssl-3.0.7-win32.zip> into a folder that's in the PATH

I need to use LuaSocket without LuaSec, because it's not fully necessary and makes a huge overhead.

- [ ] Try curl?
- [ ] Update docs
- [ ] Remove unnecessary dependencies
- [ ] Add a section to #need help for removing stuff, when erroring

**Comment by Ismoh** [comment]
There is no need to fix this debugger within lua, because the vscode debugger used for Noita is now maintained again, but within a [fork](https://github.com/Ismoh/local-lua-debugger-vscode)!

Done in https://github.com/Ismoh/noita-vscode-debugger/commit/8cda9cdef5e64eda163a398c94e43c3649308c2e

**Comment by codefaux** [comment]
>There is no need to fix this debugger within lua, because the vscode debugger used for Noita is now maintained again, but within a [fork](https://github.com/Ismoh/local-lua-debugger-vscode)!

I'm not sure what you mean by this. I've acted according to the [instructions](https://github.com/Ismoh/noita-vscode-debugger?tab=readme-ov-file#10-bullet-points-to-debug-noita) which installs the fork you mentioned, and cannot accomplish a functional debugging session.

This mod causes an error every start-up. Decoda indicates ssl.core is not provided. Noita itself simply complains that there was an error when initializing this mod. Should I unpack those random dlls to a random PATH folder? Is it meant to work without doing so? Guidance would be appreciated.

**Comment by Ismoh** [comment]
https://github.com/Ismoh/local-lua-debugger-vscode

**Comment by codefaux** [comment]
Now I'm more confused -- I just said, I followed the instructions which I linked. The instructions which I linked are from this repo. This repo's instructions link here -- https://github.com/Ismoh/local-lua-debugger-vscode/releases/latest -- IE exactly where you just linked.

I had already installed that. I tried to use it with this. It didn't work. I'm unable to intuit further instructions, and you have not written them anywhere. If that was not meant to work with this, your instructions here should not point to it. If it was, your instructions are not sufficient, thus my questions which were left ignored.

There are several "Custom Environment Examples" present on the "second" debugger page and Noita is not among them, so the instructions [here](https://github.com/Ismoh/noita-vscode-debugger?tab=readme-ov-file#10-bullet-points-to-debug-noita) are the only ones I've got.

I'm gonna need more than just a link to the thing I've already got installed. Thanks so much for your attention, the mod I'm working on has started to pick up attention and I'd really love not having to split my time between VS Code and Decoda.
